libtiff/bitarray-a1646c0/bitarray/test_bitarray.py

```diff
@@ -74,7 +76,11 @@ def assertEQUAL(self, a, b):
     def assertStopIteration(self, it):
         if is_py3k:
             return
-        self.assertRaises(StopIteration, it.__next__)
+        if is_py_pre_26:
+            self.assertRaises(StopIteration, next(it))
+        else:
+            with self.assertRaises(StopIteration):
+               next(it)
```

Comment:
Would it make more sense to just require [`unittest2`]( https://pypi.python.org/pypi/unittest2 ) on Python 2.6 and use the same syntax for all Pythons?